src/main/java/org/damap/base/rest/fits/mapper/FitsMapper.java

```diff
@@ -26,6 +26,9 @@ public Dataset mapAtoB(Fits fits, Dataset dataset) {
     dataset.setSize(getSize(fits));
     IdentificationType.Identity identity = getMajorityVoteIdentity(fits);
     dataset.setType(mapFileFormat(identity));
+    // Set file format based on mime type
```

Comment:
I checked the [RDA common standard](https://github.com/RDA-DMP-Common/RDA-DMP-Common-Standard?tab=readme-ov-file#distribution_format) and found that fileFormat should be formated like a mime type when possible. So the string splitting should be removed.

Reply:
Ah and now that i think about it, the frontend should probably give instructions to use a mime type when filling out the file format field.

Reply:
set it to the mime type for OpenAIRE mapper and FITS mapper